cloud-function/tsconfig.json

```diff
@@ -19,8 +19,7 @@
     "noPropertyAccessFromIndexSignature": true,
     "noUncheckedIndexedAccess": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "checkJs": true
```

Comment:
Curious why you removed checkJs?

Reply:
It causes typescript to attempt to overwrite ssr/dist/main.js